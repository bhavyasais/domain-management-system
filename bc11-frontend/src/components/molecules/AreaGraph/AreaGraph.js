import { Grid, Typography, withStyles } from '@material-ui/core';
import React from 'react';
import { ResponsiveContainer, AreaChart, Area, XAxis, YAxis, CartesianGrid } from 'recharts';
import PropTypes from 'prop-types';
import Dropdown from '../Dropdown/Dropdown';

const styles = () => ({
  container: {
    border: '1px solid  #eeeeee',
    padding: '24px',
    maxWidth: '769px',
  },
  header: {
    height: '40px',
  },
  textColor: {
    color: '#3e3f42',
  },
  headerContainer: {
    marginBottom: '24px',
  },
  axisStyle: {
    fontSize: '10px',
    color: '#949494',
  },
  dot: {
    width: '8px',
    height: '8px',
    borderRadius: '50%',
    // margin: '4px',
  },
});

const AreaGraph = ({ classes, header, list, selectedValue, onDropdownChange }) => {
  return (
    <Grid container className={classes.container}>
      <Grid container item direction="row" className={classes.headerContainer}>
        <Grid item container className={classes.header} xs={6} alignItems="flex-end">
          <Typography className={classes.textColor}>{header}</Typography>
        </Grid>
        <Grid item container xs={6} justify="flex-end">
          <Dropdown selectedItem={selectedValue} list={list} onChange={onDropdownChange} />
        </Grid>
      </Grid>
      {selectedValue?.data && (
        <Grid item xs={12}>
          <ResponsiveContainer width="100%" height={250}>
            <AreaChart data={selectedValue?.data} maxBarSize={50}>
              <CartesianGrid vertical={false} />
              <XAxis
                dataKey="date"
                axisLine={false}
                tickLine={false}
                className={classes.axisStyle}
                padding={{ left: 20 }}
              />
              <YAxis axisLine={false} tickLine={false} className={classes.axisStyle} />
              <Area
                dataKey="value"
                fill={selectedValue.color}
                fillOpacity={0.1}
                stroke={selectedValue.color}
                strokeWidth={2}
                dot
              />
            </AreaChart>
          </ResponsiveContainer>
        </Grid>
      )}
    </Grid>
  );
};

AreaGraph.propTypes = {
  header: PropTypes.string,
  list: PropTypes.arrayOf(
    PropTypes.shape({
      value: PropTypes.string,
      name: PropTypes.string,
      data: PropTypes.arrayOf(
        PropTypes.shape({
          date: PropTypes.string,
          value: PropTypes.string,
        }),
      ),
    }),
  ),
  selectedValue: PropTypes.shape({
    value: PropTypes.string,
    name: PropTypes.string,
    data: PropTypes.arrayOf(
      PropTypes.shape({
        date: PropTypes.string,
        value: PropTypes.string,
      }),
    ),
  }),
  onDropdownChange: PropTypes.func,
};
AreaGraph.defaultProps = {
  header: 'Default Header',
  list: [],
  selectedValue: { value: '', data: null, name: '' },
  onDropdownChange: null,
};

export default withStyles(styles)(AreaGraph);
