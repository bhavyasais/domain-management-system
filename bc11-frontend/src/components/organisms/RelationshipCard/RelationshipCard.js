import { Grid, Typography, withStyles } from '@material-ui/core';
import React from 'react';
import PropTypes from 'prop-types';
import BarGraph from '../../molecules/BarGraph/BarGraph';
import Dropdown from '../../molecules/Dropdown/Dropdown';
import clsx from 'clsx';
import { RELATIONSHIP_CARD_LEGEN_2, RELATIONSHIP_CARD_LEGEN_1 } from '../../../utils/Strings';

const styles = () => ({
  containerStyle: {
    border: 'solid 1px #eaedf3',
    margin: 0,
    width: '100%',
  },
  headerStyle: {
    borderBottom: '1px solid #eeeeee',
    width: '100%',
  },
  headerText: {
    margin: '12px 24px 12px 24px',
  },
  circle: {
    height: '8px',
    width: '8px',
    borderRadius: '50%',
    marginRight: '4px',
  },
  legendContainer: {
    paddingLeft: '24px',
  },
  dropDown: {
    marginRight: '36px',
  },
});

const RelationshipCard = ({ classes, header, children, containerStyle }) => {
  return (
    <Grid item container spacing={2} className={clsx(classes.containerStyle, containerStyle)}>
      {header && (
        <Grid item container className={classes.headerStyle} alignItems="center">
          <Typography variant="h6" className={classes.headerText}>
            {header}
          </Typography>
        </Grid>
      )}
      <Grid item container className={classes.legendContainer}>
        <Grid item container direction="row">
          <Grid
            item
            container
            direction="row"
            alignItems="center"
            xs={7}
            style={{ padding: '24px 24px 12px 24px', color: '#949494' }}
          >
            <div className={classes.circle} style={{ backgroundColor: '#ff3162' }} />
            <Typography variant="body1" style={{ marginRight: '20px' }}>
              {RELATIONSHIP_CARD_LEGEN_1}
            </Typography>
            <div className={classes.circle} style={{ backgroundColor: '#ffab2b' }} />
            <Typography variant="body1">{RELATIONSHIP_CARD_LEGEN_2}</Typography>
          </Grid>
          <Grid item container direction="row" alignItems="center" xs={5} justify="flex-end">
            <Typography variant="body1" style={{ color: '#949494' }}>
              Showing for:
            </Typography>
            <Dropdown containerStyle={classes.dropDown} />
          </Grid>
        </Grid>
      </Grid>
      <Grid
        container
        item
        direction="row"
        spacing={4}
        xs={12}
        style={{ padding: '12px 24px 12px 24px', width: '100%' }}
      >
        {children?.map((item) => (
          <Grid item container xs={6}>
            {item}
          </Grid>
        ))}
      </Grid>
    </Grid>
  );
};

RelationshipCard.propTypes = {
  header: PropTypes.string,
  children: PropTypes.arrayOf(PropTypes.element),
};
RelationshipCard.defaultProps = {
  header: null,
  children: null,
};

export default withStyles(styles)(RelationshipCard);
