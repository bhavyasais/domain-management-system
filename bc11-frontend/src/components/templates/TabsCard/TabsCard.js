import { Grid, Typography, withStyles } from '@material-ui/core';
import React, { useState } from 'react';
import PropTypes from 'prop-types';
import clsx from 'clsx';

const styles = () => ({
  headerStyle: {
    border: 'solid 1px #eaedf3',
  },
  tab: {
    height: '50px',
    color: '#949494',
    cursor: 'pointer',
    overflow: 'hidden',
  },
  activeTab: {
    color: '#1d4cd7',
    borderBottom: '2px solid #1d4cd7',
  },
});

const TabsCard = ({ children, index, tabLabels, classes, headerStyle }) => {
  const [currentIndex, setCurrentIndex] = useState(index);
  const handleChange = (idx) => {
    setCurrentIndex(idx);
  };

  return (
    <Grid container direction="column">
      <Grid item container className={clsx(classes.headerStyle, headerStyle)} direction="row">
        {tabLabels?.map((label, idx) => (
          <Grid
            item
            className={clsx(classes.tab, currentIndex === idx && classes.activeTab)}
            onClick={() => {
              handleChange(idx);
            }}
            container
            alignItems="center"
            justify="center"
            xs={1}
          >
            <Typography variant="h4">{label}</Typography>
          </Grid>
        ))}
      </Grid>
      {children?.map((item, idx) => (
        <Grid item hidden={currentIndex !== idx}>
          <div>{item}</div>
        </Grid>
      ))}
    </Grid>
  );
};

TabsCard.propTypes = {
  index: PropTypes.number,
};
TabsCard.defaultProps = {
  index: 0,
};

export default withStyles(styles)(TabsCard);
