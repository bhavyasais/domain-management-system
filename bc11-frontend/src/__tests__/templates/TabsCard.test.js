import React from 'react';
import renderer from 'react-test-renderer';
import AreaGraph from '../../components/molecules/AreaGraph/AreaGraph';
import BarGraph from '../../components/molecules/BarGraph/BarGraph';
import RelationshipCard from '../../components/organisms/RelationshipCard/RelationshipCard';
import TabsCard from '../../components/templates/TabsCard/TabsCard';

function createNodeMock() {
  const doc = document.implementation.createHTMLDocument();
  return { parentElement: doc.body };
}

describe('<SideNavigationBar/>', () => {
  let onClick = jest.fn();
  const header = 'Trustscore Trend wrt Relationship Type';
  const list = [
    {
      value: 'zemoso',
      name: 'zemoso',
      data: [
        { date: '1st Dec', value: 30 },
        { date: '2nd Dec', value: 5 },
        { date: '3rd Dec', value: 20 },
        { date: '4th Dec', value: 40 },
        { date: '5th Dec', value: 12 },
        { date: '6th Dec', value: 25 },
        { date: '7th Dec', value: 30 },
      ],
    },
    {
      value: 'stanford',
      name: 'stanford',
      data: [
        { date: '1st Dec', value: 30 },
        { date: '2nd Dec', value: 15 },
        { date: '3rd Dec', value: 12 },
        { date: '4th Dec', value: 40 },
        { date: '5th Dec', value: 20 },
        { date: '6th Dec', value: 5 },
        { date: '7th Dec', value: 30 },
      ],
    },
  ];

  const list1 = [
    {
      value: 'zemoso',
      name: 'zemoso',
      data: [
        { date: '1st Dec', value: 30 },
        { date: '2nd Dec', value: 5 },
        { date: '3rd Dec', value: 20 },
        { date: '4th Dec', value: 40 },
        { date: '5th Dec', value: 12 },
        { date: '6th Dec', value: 25 },
        { date: '7th Dec', value: 30 },
      ],
      color: '#ff3162',
    },
    {
      value: 'stanford',
      name: 'stanford',
      data: [
        { date: 'Mon', value: 30 },
        { date: 'Tue', value: 15 },
        { date: 'Wen', value: 12 },
        { date: 'Thu', value: 40 },
        { date: 'Fri', value: 20 },
        { date: 'Sat', value: 5 },
        { date: 'Sun', value: 30 },
      ],
      color: '#ff3162',
    },
  ];

  it('Renders correctly', () => {
    const wrapper = renderer.create(
      <TabsCard tabLabels={['Trust', 'Do not trust']}>
        <RelationshipCard>
          <BarGraph header="Relationship vs Group" list={list} selectedValue={list[0]} />
          <AreaGraph
            header="Trustscore Trend wrt Relationship Type"
            list={list1}
            selectedValue={list1[0]}
          />
        </RelationshipCard>
        <RelationshipCard>
          <BarGraph header="Relationship vs Group" list={list} selectedValue={list[0]} />
          <AreaGraph
            header="Trustscore Trend wrt Relationship Type"
            list={list1}
            selectedValue={list1[0]}
          />
        </RelationshipCard>
      </TabsCard>,
      { createNodeMock },
    );
    expect(wrapper).toMatchSnapshot();
  });
});
