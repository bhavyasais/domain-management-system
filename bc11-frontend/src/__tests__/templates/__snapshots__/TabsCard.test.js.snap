// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideNavigationBar/> Renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root TabsCard-headerStyle-1 MuiGrid-container MuiGrid-item"
  >
    <div
      className="MuiGrid-root TabsCard-tab-2 TabsCard-activeTab-3 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-1"
      onClick={[Function]}
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Trust
      </h4>
    </div>
    <div
      className="MuiGrid-root TabsCard-tab-2 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-1"
      onClick={[Function]}
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Do not trust
      </h4>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
    hidden={false}
  >
    <div>
      <div
        className="MuiGrid-root RelationshipCard-containerStyle-4 MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-item"
      >
        <div
          className="MuiGrid-root RelationshipCard-legendContainer-8 MuiGrid-container MuiGrid-item"
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-item"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-7"
              style={
                Object {
                  "color": "#949494",
                  "padding": "24px 24px 12px 24px",
                }
              }
            >
              <div
                className="RelationshipCard-circle-7"
                style={
                  Object {
                    "backgroundColor": "#ff3162",
                  }
                }
              />
              <p
                className="MuiTypography-root MuiTypography-body1"
                style={
                  Object {
                    "marginRight": "20px",
                  }
                }
              >
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </p>
              <div
                className="RelationshipCard-circle-7"
                style={
                  Object {
                    "backgroundColor": "#ffab2b",
                  }
                }
              />
              <p
                className="MuiTypography-root MuiTypography-body1"
              >
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </p>
            </div>
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-5"
            >
              <p
                className="MuiTypography-root MuiTypography-body1"
                style={
                  Object {
                    "color": "#949494",
                  }
                }
              >
                Showing for:
              </p>
              <div
                className="RelationshipCard-dropDown-9"
              >
                <div
                  className="MuiFormControl-root Dropdown-formControl-10"
                >
                  <div
                    className="MuiInputBase-root MuiOutlinedInput-root Dropdown-dropdown-12 MuiInputBase-formControl"
                    onClick={[Function]}
                  >
                    <div
                      aria-haspopup="listbox"
                      aria-labelledby="simple-select-label simple-select"
                      className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                      id="simple-select"
                      onBlur={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onMouseDown={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      None
                    </div>
                    <input
                      aria-hidden={true}
                      className="MuiSelect-nativeInput"
                      onAnimationStart={[Function]}
                      onChange={[Function]}
                      required={false}
                      tabIndex={-1}
                      value=""
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7 10l5 5 5-5z"
                      />
                    </svg>
                    <fieldset
                      aria-hidden={true}
                      className="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                      style={
                        Object {
                          "paddingLeft": 8,
                        }
                      }
                    >
                      <legend
                        className="PrivateNotchedOutline-legend-14"
                        style={
                          Object {
                            "width": 0.01,
                          }
                        }
                      >
                        <span
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "&#8203;",
                            }
                          }
                        />
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-grid-xs-12"
          style={
            Object {
              "padding": "12px 24px 12px 24px",
              "width": "100%",
            }
          }
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6"
          >
            <div
              className="MuiGrid-root BarGraph-container-17 MuiGrid-container"
            >
              <div
                className="MuiGrid-root BarGraph-headerContainer-20 MuiGrid-container MuiGrid-item"
              >
                <div
                  className="MuiGrid-root BarGraph-header-18 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <p
                    className="MuiTypography-root BarGraph-textColor-19 MuiTypography-body1"
                  >
                    Relationship vs Group
                  </p>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <div
                    className={null}
                  >
                    <div
                      className="MuiFormControl-root Dropdown-formControl-10"
                    >
                      <div
                        className="MuiInputBase-root MuiOutlinedInput-root Dropdown-dropdown-12 MuiInputBase-formControl"
                        onClick={[Function]}
                      >
                        <div
                          aria-haspopup="listbox"
                          aria-labelledby="simple-select-label simple-select"
                          className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                          id="simple-select"
                          onBlur={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          onMouseDown={[Function]}
                          role="button"
                          tabIndex={0}
                        >
                          zemoso
                        </div>
                        <input
                          aria-hidden={true}
                          className="MuiSelect-nativeInput"
                          onAnimationStart={[Function]}
                          onChange={[Function]}
                          required={false}
                          tabIndex={-1}
                          value="zemoso"
                        />
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M7 10l5 5 5-5z"
                          />
                        </svg>
                        <fieldset
                          aria-hidden={true}
                          className="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                          style={
                            Object {
                              "paddingLeft": 8,
                            }
                          }
                        >
                          <legend
                            className="PrivateNotchedOutline-legend-14"
                            style={
                              Object {
                                "width": 0.01,
                              }
                            }
                          >
                            <span
                              dangerouslySetInnerHTML={
                                Object {
                                  "__html": "&#8203;",
                                }
                              }
                            />
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <div
                  className="recharts-responsive-container"
                  style={
                    Object {
                      "height": 250,
                      "maxHeight": undefined,
                      "minHeight": undefined,
                      "minWidth": undefined,
                      "width": "100%",
                    }
                  }
                >
                  <div
                    style={
                      Object {
                        "display": "none",
                        "height": 0,
                        "position": "absolute",
                        "visibility": "hidden",
                        "width": 0,
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6"
          >
            <div
              className="MuiGrid-root AreaGraph-container-22 MuiGrid-container"
            >
              <div
                className="MuiGrid-root AreaGraph-headerContainer-25 MuiGrid-container MuiGrid-item"
              >
                <div
                  className="MuiGrid-root AreaGraph-header-23 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <p
                    className="MuiTypography-root AreaGraph-textColor-24 MuiTypography-body1"
                  >
                    Trustscore Trend wrt Relationship Type
                  </p>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <div
                    className={null}
                  >
                    <div
                      className="MuiFormControl-root Dropdown-formControl-10"
                    >
                      <div
                        className="MuiInputBase-root MuiOutlinedInput-root Dropdown-dropdown-12 MuiInputBase-formControl"
                        onClick={[Function]}
                      >
                        <div
                          aria-haspopup="listbox"
                          aria-labelledby="simple-select-label simple-select"
                          className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                          id="simple-select"
                          onBlur={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          onMouseDown={[Function]}
                          role="button"
                          tabIndex={0}
                        >
                          zemoso
                        </div>
                        <input
                          aria-hidden={true}
                          className="MuiSelect-nativeInput"
                          onAnimationStart={[Function]}
                          onChange={[Function]}
                          required={false}
                          tabIndex={-1}
                          value="zemoso"
                        />
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M7 10l5 5 5-5z"
                          />
                        </svg>
                        <fieldset
                          aria-hidden={true}
                          className="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                          style={
                            Object {
                              "paddingLeft": 8,
                            }
                          }
                        >
                          <legend
                            className="PrivateNotchedOutline-legend-14"
                            style={
                              Object {
                                "width": 0.01,
                              }
                            }
                          >
                            <span
                              dangerouslySetInnerHTML={
                                Object {
                                  "__html": "&#8203;",
                                }
                              }
                            />
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <div
                  className="recharts-responsive-container"
                  style={
                    Object {
                      "height": 250,
                      "maxHeight": undefined,
                      "minHeight": undefined,
                      "minWidth": undefined,
                      "width": "100%",
                    }
                  }
                >
                  <div
                    style={
                      Object {
                        "display": "none",
                        "height": 0,
                        "position": "absolute",
                        "visibility": "hidden",
                        "width": 0,
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
    hidden={true}
  >
    <div>
      <div
        className="MuiGrid-root RelationshipCard-containerStyle-4 MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-item"
      >
        <div
          className="MuiGrid-root RelationshipCard-legendContainer-8 MuiGrid-container MuiGrid-item"
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-item"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-7"
              style={
                Object {
                  "color": "#949494",
                  "padding": "24px 24px 12px 24px",
                }
              }
            >
              <div
                className="RelationshipCard-circle-7"
                style={
                  Object {
                    "backgroundColor": "#ff3162",
                  }
                }
              />
              <p
                className="MuiTypography-root MuiTypography-body1"
                style={
                  Object {
                    "marginRight": "20px",
                  }
                }
              >
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </p>
              <div
                className="RelationshipCard-circle-7"
                style={
                  Object {
                    "backgroundColor": "#ffab2b",
                  }
                }
              />
              <p
                className="MuiTypography-root MuiTypography-body1"
              >
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </p>
            </div>
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-5"
            >
              <p
                className="MuiTypography-root MuiTypography-body1"
                style={
                  Object {
                    "color": "#949494",
                  }
                }
              >
                Showing for:
              </p>
              <div
                className="RelationshipCard-dropDown-9"
              >
                <div
                  className="MuiFormControl-root Dropdown-formControl-10"
                >
                  <div
                    className="MuiInputBase-root MuiOutlinedInput-root Dropdown-dropdown-12 MuiInputBase-formControl"
                    onClick={[Function]}
                  >
                    <div
                      aria-haspopup="listbox"
                      aria-labelledby="simple-select-label simple-select"
                      className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                      id="simple-select"
                      onBlur={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onMouseDown={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      None
                    </div>
                    <input
                      aria-hidden={true}
                      className="MuiSelect-nativeInput"
                      onAnimationStart={[Function]}
                      onChange={[Function]}
                      required={false}
                      tabIndex={-1}
                      value=""
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7 10l5 5 5-5z"
                      />
                    </svg>
                    <fieldset
                      aria-hidden={true}
                      className="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                      style={
                        Object {
                          "paddingLeft": 8,
                        }
                      }
                    >
                      <legend
                        className="PrivateNotchedOutline-legend-14"
                        style={
                          Object {
                            "width": 0.01,
                          }
                        }
                      >
                        <span
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "&#8203;",
                            }
                          }
                        />
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-grid-xs-12"
          style={
            Object {
              "padding": "12px 24px 12px 24px",
              "width": "100%",
            }
          }
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6"
          >
            <div
              className="MuiGrid-root BarGraph-container-17 MuiGrid-container"
            >
              <div
                className="MuiGrid-root BarGraph-headerContainer-20 MuiGrid-container MuiGrid-item"
              >
                <div
                  className="MuiGrid-root BarGraph-header-18 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <p
                    className="MuiTypography-root BarGraph-textColor-19 MuiTypography-body1"
                  >
                    Relationship vs Group
                  </p>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <div
                    className={null}
                  >
                    <div
                      className="MuiFormControl-root Dropdown-formControl-10"
                    >
                      <div
                        className="MuiInputBase-root MuiOutlinedInput-root Dropdown-dropdown-12 MuiInputBase-formControl"
                        onClick={[Function]}
                      >
                        <div
                          aria-haspopup="listbox"
                          aria-labelledby="simple-select-label simple-select"
                          className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                          id="simple-select"
                          onBlur={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          onMouseDown={[Function]}
                          role="button"
                          tabIndex={0}
                        >
                          zemoso
                        </div>
                        <input
                          aria-hidden={true}
                          className="MuiSelect-nativeInput"
                          onAnimationStart={[Function]}
                          onChange={[Function]}
                          required={false}
                          tabIndex={-1}
                          value="zemoso"
                        />
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M7 10l5 5 5-5z"
                          />
                        </svg>
                        <fieldset
                          aria-hidden={true}
                          className="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                          style={
                            Object {
                              "paddingLeft": 8,
                            }
                          }
                        >
                          <legend
                            className="PrivateNotchedOutline-legend-14"
                            style={
                              Object {
                                "width": 0.01,
                              }
                            }
                          >
                            <span
                              dangerouslySetInnerHTML={
                                Object {
                                  "__html": "&#8203;",
                                }
                              }
                            />
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <div
                  className="recharts-responsive-container"
                  style={
                    Object {
                      "height": 250,
                      "maxHeight": undefined,
                      "minHeight": undefined,
                      "minWidth": undefined,
                      "width": "100%",
                    }
                  }
                >
                  <div
                    style={
                      Object {
                        "display": "none",
                        "height": 0,
                        "position": "absolute",
                        "visibility": "hidden",
                        "width": 0,
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6"
          >
            <div
              className="MuiGrid-root AreaGraph-container-22 MuiGrid-container"
            >
              <div
                className="MuiGrid-root AreaGraph-headerContainer-25 MuiGrid-container MuiGrid-item"
              >
                <div
                  className="MuiGrid-root AreaGraph-header-23 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <p
                    className="MuiTypography-root AreaGraph-textColor-24 MuiTypography-body1"
                  >
                    Trustscore Trend wrt Relationship Type
                  </p>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-6"
                >
                  <div
                    className={null}
                  >
                    <div
                      className="MuiFormControl-root Dropdown-formControl-10"
                    >
                      <div
                        className="MuiInputBase-root MuiOutlinedInput-root Dropdown-dropdown-12 MuiInputBase-formControl"
                        onClick={[Function]}
                      >
                        <div
                          aria-haspopup="listbox"
                          aria-labelledby="simple-select-label simple-select"
                          className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                          id="simple-select"
                          onBlur={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          onMouseDown={[Function]}
                          role="button"
                          tabIndex={0}
                        >
                          zemoso
                        </div>
                        <input
                          aria-hidden={true}
                          className="MuiSelect-nativeInput"
                          onAnimationStart={[Function]}
                          onChange={[Function]}
                          required={false}
                          tabIndex={-1}
                          value="zemoso"
                        />
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M7 10l5 5 5-5z"
                          />
                        </svg>
                        <fieldset
                          aria-hidden={true}
                          className="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                          style={
                            Object {
                              "paddingLeft": 8,
                            }
                          }
                        >
                          <legend
                            className="PrivateNotchedOutline-legend-14"
                            style={
                              Object {
                                "width": 0.01,
                              }
                            }
                          >
                            <span
                              dangerouslySetInnerHTML={
                                Object {
                                  "__html": "&#8203;",
                                }
                              }
                            />
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <div
                  className="recharts-responsive-container"
                  style={
                    Object {
                      "height": 250,
                      "maxHeight": undefined,
                      "minHeight": undefined,
                      "minWidth": undefined,
                      "width": "100%",
                    }
                  }
                >
                  <div
                    style={
                      Object {
                        "display": "none",
                        "height": 0,
                        "position": "absolute",
                        "visibility": "hidden",
                        "width": 0,
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
